### DISCUSSION TITLE
For u <3

### DISCUSSION THREAD

**Discussion by jasiukiewicztymon** [community/community#13164] [Other Feature Feedback, Questions, & Ideas] [answered]
I **Love** the new idea of for you, good luck

**Comment by thuankxk2701** [comment]
Me too!

**Comment by nonzzz** [comment]
This feature looks so cool

**Comment by OnClickListener2048** [comment]
great

**Comment by xealea** [comment, marked as answer]
This new feature is very good, you can see in detail what activities the people you follow are doing, as well as the information provided when a repo you star releases something. Nice feature

**Reply by jasiukiewicztymon** [reply]
I love the smalls colorized icons 
![image](https://user-images.githubusercontent.com/73474137/159418484-2e5220bd-708c-41f5-aae2-878f1ed38ec8.png)

**Reply by PitWenkin** [reply]
Is there any explanation what those icons mean? The yellow star does not seem to have anything to do with starred repositories …

**Reply by jasiukiewicztymon** [reply]
I have just seen two of them the star icon (the yellow one) and the release icon (the green one) and I think it will just replace the following tab with the for u propositions because it doesnt do nothing special u have just the possibility to not read the notification title
![image](https://user-images.githubusercontent.com/73474137/159432474-e162b8d3-c1cf-4b4a-a5ea-e7abe4b2ef43.png)

**Reply by PitWenkin** [reply]
I got icons on the bottom right corner of people/organisation pictures:

- green 'tag' icon, which might be related to organisations who's repositories I somehow contributed to.
- orange 'star' and blue 'person' icon, which might be people I had conversation (in isssues/pull request) with
- grey 'book' icon , which might be people for who's repositories I somehow contributed to.

I got big grey icons with recommendation:
- stars on repositories (to star them)
- person+ on people (to follw them)

**Comment by Havret** [comment]
I don't like it. It's too distracting. It's like bringing Facebook, Twitter (you name it) to your work environment.

**Comment by cap811** [comment]
Keep it, looks good.

**Comment by phipag** [comment]
Great feature!

**Comment by Redst0neFlux** [comment]
Love it

**Comment by sntwandy** [comment]
It's one of my favorite features.

**Comment by andes2912** [comment]
Cool, i like it !

**Comment by jfullstackdev** [comment]
there is a new update on the For You feed, you can now filter it, and for those who don't want it can simply disable it by unchecking all filters

![image](https://user-images.githubusercontent.com/47092464/182012117-0b19be9d-e849-462e-909e-d20d28915c1d.png)

**Reply by jasiukiewicztymon** [reply]
oh it's cool for haters ahahaha
